src/nni_manager/core/nniDataStore.ts

```diff
@@ -272,6 +284,9 @@ class NNIDataStore implements DataStore {
                     if (record.logPath !== undefined) {
                         jobInfo.logPath = record.logPath;
                     }
+                    if (jobInfo.startTime === undefined && record.timestamp !== undefined) {
```

Comment:
would be better to also add brief comment here

Reply:
Thanks, updated.